docs/docs/gatsby-cli.md

```diff
@@ -200,30 +202,30 @@ You can type in a command, such as one of these:
 
 `staticQueries`
 
-When combined with the [GraphQL explorer](/docs/introducing-graphiql/), these REPL commands could be very helpful for understanding your Gatsby site's data.
+[GraphQL explorer](/docs/introducing-graphiql/) के साथ उपयोग करने पर, ये आरईपीएल कमांड आपके Gatsby साइट के डेटा को समझने के लिए बहुत सहायक हो सकते हैं।
 
-For more information, check out the [Gatsby REPL documentation](/docs/gatsby-repl/).
+अधिक जानकारी के लिए, देखें [Gatsby REPL डॉक्युमेंटेशन](/docs/gatsby-repl/)
 
-### Disabling colored output
+### कलर आउटपुट को डिसेबल करना
 
-In addition to the explicit `--no-color` option, the CLI respects the presence of the `NO_COLOR` environment variable (see [no-color.org](https://no-color.org/)).
+सीएलआई ओप्शंस `--no-color` के अलावा एन्वाइरन्मेंट वेरियबल `NO_COLOR` की उपस्थिति का सम्मान करता है (देखें [no-color.org](https://no-color.org/)).
 
-## How to change your default package manager for your next project?
+## अपने अगले प्रोजेक्ट के लिए अपने डिफ़ॉल्ट पैकेज मैनेजर को कैसे बदलें?
 
-When you use `gatsby new` for the first time to create a new project, you are asked to choose your default package manager between yarn and npm.
+जब आप पहली बार एक नया प्रोजेक्ट बनाने के लिए `gatsby new` का उपयोग करते हैं, तो आपको yarn और npm के बीच अपने डिफ़ॉल्ट पैकेज मैनेजर को चुनने के लिए कहा जाता है।
 
 ```shell
 Which package manager would you like to use ? › - Use arrow-keys. Return to submit.
 ❯  yarn
    npm
 ```
 
-Once you’ve made your choice, the CLI won’t ask for your preference again for any subsequent project.
+एक बार जब आप अपनी पसंद बना लेते हैं, तो सीएलआई किसी भी बाद की प्रोजेक्ट के लिए आपकी प्राथमिकता के लिए नहीं पूछेगा।
 
-If you want to change this for your next project you have to edit the config file created automatically by the CLI.
-This file is available on your system at: `~/.config/gatsby/config.json`
+यदि आप इसे अपनी अगली प्रोजेक्ट के लिए बदलना चाहते हैं, तो आपको सीएलआई द्वारा स्वचालित रूप से बनाई गई कॉन्फ़िगर फ़ाइल को  एडिट करना होगा।
+यह फ़ाइल आपके सिस्टम पर उपलब्ध है: `~/.config/gatsby/config.json`
 
-In it you’re going to see something like this.
+इसमें आप कुछ इस तरह से देखने वाले हैं।
```

Comment:
`देखने वाले हैं` => `दिखने वाला है`

thoughts on this?
Which one sounds better?

Reply:
```इसमें आप कुछ इस तरह से दिखने वाला है।``` is not correct.
between below two, first one makes more sense to me.
```इसमें आप कुछ इस तरह से देखने वाले हैं।'''
OR
```इसमें कुछ इस तरह से दिखने वाला हैं।```